src/main/java/core/basesyntax/Circle.java

```diff
@@ -0,0 +1,24 @@
+package core.basesyntax;
+
+public class Circle extends Figure {
+    private double radius;
+
+    public Circle(String color, double radius) {
+        super(color);
+        this.radius = radius;
+    }
+
+    @Override
+    public double calculateArea() {
+        double area = Math.PI * Math.pow(radius, 2);
+        return Math.round(area);
+    }
+
+    @Override
+    public void draw() {
+        System.out.println("Figure: circle,"
+                + " area: " + calculateArea()
+                + " sq. units, radius: " + radius
+                + " units, color: " + getColor());
```

Comment:
The code within the `draw` method is using string concatenation within the `System.out.println` call. This is less efficient than using a `StringBuilder` or simply separating the strings and variables with commas to utilize the `println` method's built-in string conversion.